src/components/pages/login/LoginForm.jsx

```diff
@@ -1,5 +1,12 @@
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+import { IoChevronForward } from "react-icons/io5"
+import { BsPersonCircle } from "react-icons/bs"
+import Input from "../../reusable-ui/TextInput";
```

Comment:
```suggestion
import TextInput from "../../reusable-ui/TextInput";
```

Par souci de clarté et de cohérence, il faut conserver le même nom de composant au moment de l'import